lib/kodi/utils/Time.h

```diff
@@ -0,0 +1,91 @@
+#pragma once
```

Comment:
Don't add the Time.h, thought on begin to use it on Kodi's addon headers but with the new C++11 functionality not needed anymore.